tests/kuttl/tests/fernet_rotation/03-assert.yaml

```diff
@@ -0,0 +1,6 @@
+# validate that token saved in previous step works
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+namespaced: true
+commands:
+  - script: ../../common/scripts/test_invalid_token.sh
```

Comment:
The file doesn't exit, should it be `validate_test_token.sh` instead?

Reply:
Thanks, it didn't go with `git add .`
Just submitted a new patch with that instead.
That's a different script, while `validate_test_token.sh` expects the token to work, `test_invalid_token.sh` does the same but the token should fail.

Reply:
Oh okay, thanks for clarifying that!